CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+* [percona/azure_exporter](https://github.com/percona/azure_metrics_exporter)
+* [Percona-Lab/clickhouse_exporter](https://github.com/Percona-Lab/clickhouse_exporter)
+* [percona/percona-toolkit](https://github.com/percona/percona-toolkit)
+
+### Building and Packaging
+
+* [percona/pmm-server](https://github.com/percona/pmm-server)
+* [Percona-Lab/pmm-submodules](https://github.com/Percona-Lab/pmm-submodules)
+* [Percona-Lab/jenkins-pipelines](https://github.com/Percona-Lab/jenkins-pipelines)
+
+### QA, Testing and Documentation
+* [percona/pmm-ui-tests](https://github.com/percona/pmm-ui-tests)
+* [percona/pmm-qa](https://github.com/percona/pmm-qa)
+* [percona/pmm-doc](https://github.com/percona/pmm-doc)
+
+## APIs
+
+See API definitions [here](https://percona-pmm.readme.io/reference/introduction).
+
+## Prerequisites
+
+Before submitting code or documentation contributions, you should first complete the following prerequisites.
+
+
+
+### 1. Sign the CLA
+
+Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/<linktoCLA>) (CLA). You can do this using your GitHub account and one click.
+
+### 2. Code of Conduct
+
+Please make sure to read and agree to our [Code of Conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
+
+## Submitting a Bug
+
+If you find a bug in Percona Monitoring and Management  or one of the related projects, you should submit a report to that project's [JIRA](https://jira.percona.com) issue tracker. Some of related project also have GitHub Issues enabled, so you also could submit there.
+
+Your first step should be [to search](https://jira.percona.com/issues/?jql=project=PMM) the existing set of open tickets for a similar report. If you find that someone else has already reported your problem, then you can upvote that report to increase its visibility.
+
+If there is no existing report, submit a report following these steps:
+
+1. [Sign in to Percona JIRA.](https://jira.percona.com/login.jsp) You will need to create an account if you do not have one.
+2. [Go to the Create Issue screen and select the relevant project.](https://jira.percona.com/secure/CreateIssueDetails!init.jspa?pid=11600&issuetype=1&priority=3)
+3. Fill in the fields of Summary, Description, Steps To Reproduce, and Affects Version to the best you can. If the bug corresponds to a crash, attach the stack trace from the logs.
+
+An excellent resource is [Elika Etemad's article on filing good bug reports.](http://fantasai.inkedblade.net/style/talks/filing-good-bugs/).
+
+As a general rule of thumb, please try to create bug reports that are:
+
+- *Reproducible.* Include steps to reproduce the problem.
+- *Specific.* Include as much detail as possible: which version, what environment, etc.
+- *Unique.* Do not duplicate existing tickets.
+
+## Setup your local development environment
+<explanation of local setup>
```

Comment:
here we can say that user can clone pmm-managed, run `make env-up` to start PMM Server and then clone `pmm-agent` and run `make setup-dev` there, so he will have a ready environment with minimum stuff required.